templates/TIR.html.tmpl

```diff
@@ -203,6 +203,52 @@
               </table>
             </tr>
           {{/builds}}
+        {{/openShiftData}}
+        {{#if deployment}}
+        {{#deployment}}
+            <tr>
+              <table>
+              {{#status}}
+              <tr>
+                  <b>Deployment Status</b>
+                  <table>
+                    <thead>
+                        <th>Name</th>
+                        <th class="content-wrappable">Value</th>
+                    </thead>
+                    <tbody>
+                        {{#each .}}
+                          <tr>
+                            <td>{{@key}}</td><td class="content-wrappable">{{this}}</td>
```

Comment:
If not mistaken, the information shown in this table is not dynamic. The concrete rows and their keys are fixed. Moreover, we want to show text that is meaningful for a validation manager or an auditor. "helmEnvBasedValuesFiles" is not understandable by a validation manager. The place to give the human readable version of this information is not the shared library, but the template. This should be modified to show a fixed set of rows with human-friendly names.

Reply:
This logic has been replaced by one that generates human friendly table rows.